pages/api/calendar.js

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon';
+
+import { getTalks } from '../../lib/airtable';
+
+const currentDate = DateTime.local();
+
+const removeOldTalks = talk => new Date(talk.date) > new Date();
```

Comment:
Maybe rename it `isTalkPassed` or something like it, because it doesn't actually remove an old talk